packages/@sanity/components/src/tags/story.js

```diff
@@ -61,9 +46,8 @@ storiesOf('Tags')
         <TagsTextField
           label={text('label (prop)', 'Tags')}
           placeholder={text('placeholder (prop)', 'This is the placeholder')}
-          tags={array('tags (prop)', tags)}
-          onAddTag={action('onAddTag')}
-          onRemoveTag={action('onRemoveTag')}
+          value={array('tags (prop)', tags)}
```

Comment:
The prop is now named value. Please change the description from `tags (prop)` to `value (prop)`.  There should also be an action. In this case `onAddTag` is removed. Maybe there should be an `onChange`?

Reply:
Changed to `value`.

Not sure what you mean by

> There should also be an action. In this case onAddTag is removed. Maybe there should be an onChange?

(there's an `onChange` on the line below value)